src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -44,15 +54,10 @@ public ScannerResultControl()
         {
             InitializeComponent();
             _list = new List<ScanListViewItemViewModel>();
-            lvDetails.AddHandler(Thumb.DragDeltaEvent, new DragDeltaEventHandler(Thumb_DragDelta), true);
+            listControl.AddHandler(Thumb.DragDeltaEvent, new DragDeltaEventHandler(Thumb_DragDelta), true);
```

Comment:
For the AutomatedChecks work, the `DragDeltaEvent` method went into the child, then we added the handler during the child's InitializeComponents. Might be worth taking the same approach here